tests/Integration/Behaviour/Features/Scenario/CustomerService/customer_service_summary.feature

```diff
@@ -0,0 +1,15 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s customer_service --tags customer-management
+@restore-all-tables-before-feature
+@customer-management
+Feature: Customer service summary
+
+  Scenario: Get default customer service summary
+    Given that default contacts exists
```

Comment:
SAme here you can create your own contact instead of relying on existing ones and explicit IDs